src/vivarium_cluster_tools/psimulate/worker/vivarium_work_horse.py

```diff
@@ -133,6 +77,90 @@ def work_horse(job_parameters: dict) -> tuple[pd.DataFrame, dict[str, pd.DataFra
         logger.info(f"Exiting job: {job_parameters}")
 
 
+def get_sim_results(
```

Comment:
Nit: can you put these functions in the same order they're referenced in the above method?